docs/generators/changelog/versions/2.13.md

```diff
@@ -26,12 +26,29 @@ TODO
 #### Meteor Version Release
 
 
+
+* `ddp-server@get-version`:
+    - Updated livedata server test to be more easily debbuged.
```

Comment:
```suggestion
    - Updated livedata server test to be more easily debugged.
```

Also, this change doesn't affect Meteor apps or packages using ddp-server. Is there something we can do to to make it easier for developers to see what changes are relevant? For example, consistently move these types of changes to the end, or add a new `chore` section with these?

For this release there aren't many of these types of changes, but some past releases had a large number and it made it more difficult to understand what changed.

Reply:
We have the internal API changes, I was thinking in turning into something like that, just internal changes